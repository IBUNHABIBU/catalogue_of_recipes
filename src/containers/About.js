import React from 'react';
import { Typography, Container } from '@material-ui/core';
import useStyles from '../stylesheets/styles';

const About = () => {
  const classes = useStyles();
  return (
    <Container maxWidth="sm" className={classes.container}>
      <Typography variant="h5" component="h2">
        Learn different food recipes via this website
      </Typography>
      <Typography variant="body1" component="h2" gutterBottom>
        All data in these website come from The MealDb api
      </Typography>
    </Container>
  );
};

export default About;
