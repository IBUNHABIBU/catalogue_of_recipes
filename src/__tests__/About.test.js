import React from 'react';
import { shallow } from 'enzyme';
import About from '../containers/About';
import { Provider } from 'react-redux'

const setup = (props={}) => {
  const component = shallow(<About {...props} />);
  return component;
}

describe('Testing the About page', () => {
  let component;
  beforeEach(() => {
    component = setup();
  })
  it('should display the about component with className container', () => {
    const wrapper = component.find('.container');
    expect(wrapper.length).toBe(1);
  });
});
