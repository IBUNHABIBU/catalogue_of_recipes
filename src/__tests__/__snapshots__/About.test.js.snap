// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders App component correctly 1`] = `
<div
  className="container about"
>
  <p>
    Learn different food recipes via this website
  </p>
  <p>
    All data in these website come from The MealDb api
  </p>
</div>
`;
